### PR TITLE
fix getSqlText() undefined on multi statement for issue 187

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-dl**
https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/187
not return getSqlText() on multi statement

### CODE REVIEW

File: lib/connection/statement.js [hunk 1]
```diff
@@ -688,6 +691,8 @@ function BaseStatement(
           Util.format('request id = %s', context.requestId) :
           Util.format('statement id = %s', context.statementId));
     }
+    this.hasNext = hasNextResult(this, context);
+    this.NextResult = createNextReuslt(this, context);
```
Comment (sfc-gh-igarish) [inline lib/connection/statement.js:695]: Typo

File: lib/connection/statement.js [hunk 2]
```diff
@@ -824,6 +832,7 @@ function createOnStatementRequestSuccRow(statement, context)
         context.isMulti = true;
         context.multiResultIds = this._resultIds;
         context.multiCurId = 0;
+        context.arrSqlText = context.sqlText.split(';');
```
Comment (sfc-gh-igarish) [inline lib/connection/statement.js:835]: will this work in all cases? do we support store proc creation as multi statement? if yes,  one store proc definition can contains many semicolons. Also how about a comment contains a semi colon?

Comment (sfc-gh-ext-simba-dl) [reply lib/connection/statement.js:835]: should we always return the whole multistatements for each sub statement?

Comment (sfc-gh-ext-simba-dl) [reply lib/connection/statement.js:850]: I added code cover the store proc

**Comment by sfc-gh-dszmolka** [comment]
fixed on another PR

**Pull request closed without merging**